Revises index, indexes (#5784)
opensearch-project/documentation-website: TERMS.md

**index, indexes**

> In technical documentation and the UI, use *indexes* as the plural form of *index*. Use *indices* only in the context of mathematical expressions. Non-hardcoded references to *indices* should be changed to *indexes*.

Reviewer: ```suggestion
In technical documentation and the UI, use *indexes* as the plural form of *index*. Use *indices* only in the context of mathematical expressions. Variable and setting names should not be changed.
```
